src/label-helpers.ts

```diff
@@ -22,37 +25,46 @@ function getTextContent(node) {
     .join('')
 }
 
-function getLabelContent(node) {
+function getLabelContent(node: Node | Element | HTMLInputElement) {
   let textContent
-  if (node.tagName.toLowerCase() === 'label') {
+  if ('tagName' in node && node.tagName.toLowerCase() === 'label') {
```

Comment:
```suggestion
  if (node.tagName?.toLowerCase() === 'label') {
```

Does this work?

Reply:
Shouldn't we make sure we don't pass a `Node` here?